cmd/monitor/streaming/bootstrap.yaml

```diff
@@ -521,6 +521,64 @@ notify-storage:
       auto.offset.reset: "${KAFKA_AUTO_OFFSET_RESET:latest}"
       auto.commit.interval.ms: "${KAFKA_AUTO_COMMIT_INTERVAL_MS:1000}"
 
+# --- data pipeline block ---
+erda.oap.collector.core:
+  pipelines:
+    spans:
+      - receivers:
+          - "erda.oap.collector.receiver.kafka@spotspan"
+          - "erda.oap.collector.receiver.kafka@oapspan"
+        processors:
+          - "erda.oap.collector.processor.modifier@span"
+        exporters:
+          - "erda.oap.collector.exporter.clickhouse@span"
+        batch_size: 1000
+        _enable: ${WRITE_SPAN_TO_CLICKHOUSE_ENABLE:false}
+
+# ************* receivers *************
+erda.oap.collector.receiver.kafka@spotspan:
+  _enable: ${WRITE_SPAN_TO_CLICKHOUSE_ENABLE:false}
+  proto_parser: spotspan
+  consumer:
+    topics: "${SPOT_TRACE_TOPICS:spot-trace}"
+    group: "${TRACE_GROUP_ID:spot-monitor-trace-dev}"
+    parallelism: ${SPOT_SPOTSPAN_CONSUMERS:9}
+    options:
+      auto.offset.reset: "${KAFKA_AUTO_OFFSET_RESET:latest}"
+      auto.commit.interval.ms: "${KAFKA_AUTO_COMMIT_INTERVAL_MS:1000}"
+
+erda.oap.collector.receiver.kafka@oapspan:
+  _enable: ${WRITE_SPAN_TO_CLICKHOUSE_ENABLE:false}
+  proto_parser: oapspan
+  consumer:
+    topics: "${OAP_TRACE_TOPICS:msp-jaeger-trace,erda-spans}"
+    group: "${TRACE_GROUP_ID:spot-monitor-trace-dev}"
+    parallelism: ${SPOT_OAPSPAN_CONSUMERS:9}
+    options:
+      auto.offset.reset: "${KAFKA_AUTO_OFFSET_RESET:earliest}"
+      auto.commit.interval.ms: "${KAFKA_AUTO_COMMIT_INTERVAL_MS:1000}"
+# ************* receivers *************
+
+# ************* processors *************
+erda.oap.collector.processor.modifier@span:
+  _enable: ${WRITE_SPAN_TO_CLICKHOUSE_ENABLE:false}
+  rules:
+    - action: drop
+      key: tags.http_read_bytes
+    - action: drop
+      key: tags.http_request_content_length
+    - action: drop
+      key: tags.http_wrote_bytes
+# ************* processors *************
+
+# ************* exporters *************
+erda.oap.collector.exporter.clickhouse@span:
+  _enable: ${WRITE_SPAN_TO_CLICKHOUSE_ENABLE:false}
+  currency_num: ${EXPORTER_CLICKHOUSE_CURRENCY_NUM:30}
```

Comment:
30 may be too large for the clickhouse, as clickhouse expect less concurrency writer with large size batch. use 3 as default

Reply:
ok, already default to 3